projects/vdk-core/src/vdk/internal/builtin_plugins/run/file_based_step.py

```diff
@@ -74,8 +74,27 @@ def run(job_input)
             sys.path.insert(0, str(step.job_dir))
             filename = step.file_path.name
             namespace = "step_" + str(filename).lower().strip(".py")
-            python_module = imp.load_source(namespace, str(step.file_path))
             success = False
+
+            try:
+                log.debug("Loading %s ..." % filename)
```

Comment:
nit: Why not using f strings?

Reply:
for consistency with the same usage, several lines bellow:
`log.info("Entering %s#run(...) ..." % filename)`

Reply:
Gotcha, ok.

Reply:
Note that some of those lines were written 4 years ago or more before "f-strings". So we should switch them all over to f-strings when we see opportunity.